src/collections/_documentation/platforms/android/index.md

```diff
@@ -0,0 +1,597 @@
+
+If there is a fatal error in your native code, the process is the same. The difference is, a restart is always needed, as a hard crash immediately stops the app.
+
+The NDK is not only catching the unhandled exceptions but is also set as a signal handler to be able to react to the signals from OS. When the application is about to crash, an error report is created and saved to disk. The SDK will try to send the report right after the crash, but since the environment may be unstable at the crash time, the report is guaranteed to send once the application is started again.
+
+### Capturing Errors manually
+
+Insight into the health of your application doesn't require crashes. You can report exceptions and messages manually using our API.
+
+```java
+public class DemoClass {
+    /**
+        * An example method that throws an exception.
+        */
+    public void unsafeMethod() {
+        throw new RuntimeException();
+    }
+
+    /**
+        * An Example of how to report an error and how to enrich the error context.
+        */
+    void logWithStaticAPI() {
+    /*
+    Record a breadcrumb in the current context, which will be sent
+    with the next event(s). By default, the last 100 breadcrumbs are kept.
+    */
+        Breadcrumb breadcrumb = new Breadcrumb();
+        breadcrumb.setMessage("User made an action.");
+        Sentry.addBreadcrumb("the user performed an action");
+
+        // Set the user in the current context.
+        Sentry.configureScope(
+            scope -> {
+                User user = new User();
+                user.setEmail("hello@sentry.io");
+                scope.setUser(user);
+            });
+
+    /*
+    This sends a simple event to Sentry, with a simple String message.
+    */
+
+        Sentry.captureMessage("This is a test");
+
+        try {
+            unsafeMethod();
+        } catch (Exception e) {
+        // This sends an error report to the  Sentry
+        Sentry.captureException(e);
+        }
+    }
+}
+```
+
+## ProGuard
+
+To use ProGuard with Sentry, upload the ProGuard mapping files by using our Gradle integration (**recommended**) or manually by using [sentry-cli]({%- link _documentation/cli/dif.md -%}#proguard-mapping-upload).
+
+### Gradle Integration
+
+Using Gradle (Android Studio) in your `app/build.gradle` add:
+
+```groovy
+apply plugin: 'io.sentry.android.gradle'
+```
+
+And declare a dependency in your top-level `build.gradle`:
+
+```groovy
+buildscript {
+    dependencies {
+        classpath 'io.sentry:sentry-android-gradle-plugin:1.7.28'
+    }
+}
+```
+
+The plugin will automatically generate appropriate ProGuard mapping files and upload them when you run `gradle` `assembleSomething`. For example, `assembleRelease` — Release is the default, but the plugin works for others if you have enabled ProGuard. The credentials for the upload step are loaded via environment variables.
```

Comment:
what's about `assemble{BuildVariant}` instead of `assembleSomething`?

Reply:
@marandaneto which is correct, 1 or 2:
1.  `gradle` `assemble{BuildVariant}`
2. `gradle assemble{BuildVariant}`